04-vector-raster-integration-advanced.Rmd

```diff
@@ -298,46 +322,49 @@ CHM_HARV_manualCrop <- crop(x = chm_HARV, y = new.extent)
 #plot extent boundary and newly cropped raster
 plot(aoiBoundary_HARV, 
      main = "Manually Cropped Raster\n NEON Harvard Forest Field Site")
-plot(new.extent, col="blue", lwd=4,add = TRUE)
+plot(new.extent, col="red", lwd=4,add = TRUE)
 plot(CHM_HARV_manualCrop, add = TRUE)
 
 ```
 
-Notice that our manual `new.extent` is smaller than the `aoiBoundary_HARV` 
-and that the raster is now the same as the `new.extent` object.
+Notice that our manual `new.extent` (in red) is smaller than the
+`aoiBoundary_HARV` and that the raster is now the same as the `new.extent`
+object.
  
 See the documentation for the `extent()` function for more ways
 to create an `extent` object. 
 
-* `help.search("extent", package = "raster"))
+* `help.search("extent", package = "raster")`
 * More on the 
 <a href="http://www.inside-r.org/packages/cran/raster/docs/extent" target="_blank">
 extent class in `R`</a>.
 
-##Extract Raster Pixels Values Using Vector Polygons
 
+##Extract Raster Pixels Values Using Vector Polygons
 Often we want to extract values from a raster layer for particular locations - 
 for example, plot locations that we are sampling on the ground. 
 
-To do this in `R`, we use the `extract()` function. We will begin by extracting 
-all canopy height pixel values located within our `aoiBoundary` polygon which
-surrounds the tower located at the NEON Harvard Forest field site. The `extract()`
-function requires:
-
-* The raster that you wish to extract values from.
-* The vector layer containing the polygons that you wish to use as a boundary or 
-boundaries.
-* OPTIONALLY: you can tell it to store the output values in a `data.frame` using
-`df=TRUE`.
-
 <figure>
     <a href="http://neondataskills.org/images/spatialData/BufferSquare.png">
     <img src="http://neondataskills.org/images/spatialData/BufferSquare.png"></a>
-    <figcaption> Extract raster information using a polygon boundary. 
+    <figcaption> Extract raster information using a polygon boundary. We can
+    extract all pixel values within 20m of our x,y point of interest. These can 
+    then be summarized into some value of interest (e.g. mean, maximum, total).
     Source: National Ecological Observatory Network (NEON).  
     </figcaption>
 </figure>
 
+To do this in `R`, we use the `extract()` function. We will begin by extracting 
+all canopy height pixel values located within our `aoiBoundary` polygon which
+surrounds the tower located at the NEON Harvard Forest field site. The
+`extract()` function requires:
+
+* The raster that we wish to extract values from,
+* The vector layer containing the polygons that we wish to use as a boundary or 
+boundaries,
+* we can tell it to store the output values in a `data.frame` using
+`df=TRUE` (optional, default is to NOT return a `data.frame`) .
+
```

Comment:
@mjones01 did you view this page as built before pushing? when you have a unordered list before a code chunk it causes render problems. that is why i had things in the other order.